Resources/doc/cookie_sync.rst

```diff
@@ -0,0 +1,177 @@
+Synchronizing Cart And Customer
+===============================
+
+Synchronizing from Magento to ONGR is done via cookies. After every action with cart (adding or removing products)
+a cookie is set with all the products in cart. Also after login, logout and customer information update another cookie
+is updated.
+
+Synchronization from ONGR to Magento is done via requests. More detailed information about will come up later.
+
+Setting up synchronization will require installing `MagentoSyncModule <https://github.com/ongr-io/MagentoSyncModule>`_
+and a bit of configuration.
+
+Step 1. Installing Magento Module
+---------------------------------
+
+Follow standard procedure for installing `Magento Sync Module <https://github.com/ongr-io/MagentoSyncModule>`_. Then,
+if needed, set domain of ONGR project in ``system->configuration->ONGR SYNC->Sync Options``. This setting is used for
+cookie domain and if left empty current domain will be used.
```

Comment:
there's something wrong with this sentence. Let's try "This settign is used for cookie domain. If it is left empty, current domain will be used."